cmrtlib/linux/CMakeLists.txt

```diff
@@ -90,8 +91,26 @@ endif()
 set_target_properties( igfxcmrt PROPERTIES 
   FOLDER CM_RT 
   COMPILE_DEFINITIONS "${CMRT_DEFINES}" 
-  OUTPUT_NAME "igfxcmrt${TARGET_MODIFIER}"
+  OUTPUT_NAME "libigfxcmrt${TARGET_MODIFIER}"
   )
+
+# set the version, aligned with media driver
+if(NOT DEFINED MEDIA_VERSION OR MEDIA_VERSION STREQUAL "")
+    set(MEDIA_VERSION "1.0.0")
+endif()
+set(CMRT_VERSION ${MEDIA_VERSION})
+string(REPLACE "." ";" CMRT_VERSION_LIST ${CMRT_VERSION})
+list(GET CMRT_VERSION_LIST 0 CMRT_VERSION_MAJOR)
+
+set_target_properties(igfxcmrt PROPERTIES VERSION ${CMRT_VERSION})
+set_target_properties(igfxcmrt PROPERTIES SOVERSION ${CMRT_VERSION_MAJOR})
 set_target_properties( igfxcmrt PROPERTIES PREFIX "")
 target_link_libraries( igfxcmrt dl va rt ${GCC_SECURE_LINK_FLAGS})
 
+ADD_CUSTOM_TARGET(link_target ALL COMMAND ${CMAKE_COMMAND} -E create_symlink libigfxcmrt${TARGET_MODIFIER}.so igfxcmrt${TARGET_MODIFIER}.so)
+
+if(DEFINED CMAKE_INSTALL_LIBDIR AND (NOT CMAKE_INSTALL_LIBDIR STREQUAL ""))
```

Comment:
These checks are absolutely useless and should be dropped. CMAKE_INSTALL_LIBDIR _will_ be defined once we include GNUInstallDirs which we do unconditionally here:
https://github.com/intel/media-driver/blob/f9b1c19c36ead3cca95295581f0fee8b8b99a137/CMakeLists.txt#L43
If user did not specify CMAKE_INSTALL_LIBDIR, then system default will be used. So, drop the condition, please.

Reply:
This CMakeFile can be used directly sometime, like by the build_linux.sh. This is what we need to generate the CM SDK package.